cloudlift/deployment/cluster_template_generator.py

```diff
@@ -525,10 +525,10 @@ def _add_ec2_auto_scaling(self):
                                 '\n'.join([
                                 'echo ECS_CLUSTER=${Cluster} >> /etc/ecs/ecs.config',
                                 'echo ECS_RESERVED_MEMORY=256 >> /etc/ecs/ecs.config',
-                                'echo ECS_AVAILABLE_LOGGING_DRIVERS=[\"awslogs\",\"fluentd\"]  >> /etc/ecs/ecs.config',
+                                'echo ECS_AVAILABLE_LOGGING_DRIVERS=["awslogs","fluentd"]  >> /etc/ecs/ecs.config',
```

Comment:
```python
"[\"awslogs\",\"fluentd\"]" == '["awslogs","fluentd"]'
# True
``` 
these two are equivalent in python, and also in shell it echoes out without double quotes.
```sh
echo ["awslogs","fluentd"]
# echoes [awslogs,fluentd]
```
So we need 
```sh
echo [\"awslogs\", \"fluentd\"] 
# echoes ["awslogs", "fluentd"] which is a valid JSON 
```
To get it, we need to escape backslash and quotation mark
```python
'echo ECS_AVAILABLE_LOGGING_DRIVERS=[\\\"awslogs\\\", \\\"fluentd\\\"] >> /etc/ecs/ecs.config',
```
https://github.com/GetSimpl/cloudlift/blob/db663f1fae4d09ba5132826f7f6517c0c44a8ca0/cloudlift/deployment/cluster_template_generator.py#L528

Reply:
Good catch. Adding to this,
replacing the current code with `'echo ECS_AVAILABLE_LOGGING_DRIVERS=\'["awslogs","fluentd"]\'  >> /etc/ecs/ecs.config'` might be simpler.

> cat /etc/ecs/ecs.config
ECS_CLUSTER=cluster-testing15
ECS_RESERVED_MEMORY=256
ECS_AVAILABLE_LOGGING_DRIVERS=["awslogs","fluentd"]
ECS_INSTANCE_ATTRIBUTES={"deployment_type": "ondemand"}

Reply:
https://github.com/GetSimpl/cloudlift/pull/134/commits/110e318641ce12de9d08eed606dad2840ae2274f